suggestion.txt

```diff
@@ -0,0 +1,69 @@
+ReactOS Wallpapers suggestion:
```

Comment:
(Almost?) all the links in this document seem dead, that's why I'm not fully sure it's useful to keep it alive.
For example, the "ReactOS Wallpapers (earth)" was linking to this page:
https://web.archive.org/web/20060925235553/http://reactosde.re.funpic.de/mediadetail.php?sec=Graphic%20arts&cat=wallp&mdnr=36

At most I think we could just have one unique link to the webarchived website to the wallpapers section:
https://web.archive.org/web/20060925223103/http://reactosde.re.funpic.de/medialist.php?sec=Graphic%20arts&cat=wallp

Reply:
Fixed.